src/command_modules/azure-cli-extension/azure/cli/command_modules/extension/__init__.py

```diff
@@ -55,6 +55,8 @@ def load_arguments(self, command):
             c.argument('extension_name', options_list=['--name', '-n'], help='Name of extension', completer=extension_name_completion_list)
             # This is a hidden parameter for now
             c.argument('index_url', options_list=['--index'], help=argparse.SUPPRESS)
+            c.argument('pip_proxy', options_list=['--pip-proxy', '-pp'],
```

Comment:
Short arguments can only have a single character after the -.
I would just remove the `-pp` or you can use `-p` is that's not currently used.

Reply:
Updated. But did you mean convention wise in this command module? We are using multi-character short arguments in the IoT extension and they seem to be working fine.